index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kartica</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <img class="top-pattern" src="./slike/bg-pattern-top.svg" alt="pattern1">
+    <img class="bottom-pattern" src="./slike/bg-pattern-bottom.svg" alt="pattern2">
+    <div class="main">
+        <div class="card">
+            <div class="card-section-one">
+                <img src="./slike/bg-pattern-card.svg" alt="">
+            </div>
+            <div class="card-section-two">
+                <img class="rounded-image" src="./slike/image-victor.jpg" alt="profile1">
+                <p><b>Victor Crest</b>&nbsp;&nbsp;<span style="color:gray">26</span></p>
+                <p><span style="color:gray;font-size: 14px;">London</span></p>
```

Comment:
Ne moras da pravis nove commitove, ovo je generalno ok, ali moja sugestija je da ne koristis inline CSS, osim u nekim situacijama kada je bas apsolutno neophodno, a ovde to nije slucaj